src/pages/en/guides/typescript.md

```diff
@@ -102,7 +102,7 @@ const { greeting = 'Hello', name } = Astro.props as Props;
 
 Astro provides JSX type definitions to check that your markup is using valid HTML attributes. You can use these types to help build component props. For example, if you were building a `<Link>` component, you could do the following to mirror the default HTML attributes in your component’s prop types.
 
-```astro title="src/components/Link.astro"
+```astro title="src/components/Link.astro" {2} "as Props"
```

Comment:
Same applies here

Reply:
I agree to both of your suggestions! Will change that :)